client/me/purchases/payment-methods/payment-method-backup-toggle.tsx

```diff
@@ -82,6 +83,11 @@ export default function PaymentMethodBackupToggle( { card }: { card: StoredCard
 								showText={ false }
 								supportContext="payment_methods_manage"
 								iconSize={ 16 }
+								supportLink={
+									isJetpackCloud()
+										? 'https://wordpress.com/support/payment/#manage-payment-methods'
```

Comment:
Underlying modal is not implemented in cloud. Link to support page instead.

Reply:
There's a new link you could use here that provides some more detail; see https://github.com/Automattic/wp-calypso/pull/59330